lib/plugins/aws/provider.js

```diff
@@ -828,6 +828,7 @@ class AwsProvider {
                     {
                       type: 'object',
                       properties: {
+                        name: { type: 'string' },
```

Comment:
Let's ensure a bit better validation here following the valid values for RoleName: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html#cfn-iam-role-rolename

Reply:
:white_check_mark: